test/unit/cpu/test-nested.cpp

```diff
@@ -103,19 +103,19 @@ using TRANSFORMS =
 
 template <typename TRANSFORMS>
 using POLICIES =
-    std::tuple<ExecInfo<TRANSFORMS, seq_exec, seq_exec>,
-               ExecInfo<TRANSFORMS, seq_exec, simd_exec>,
-               ExecInfo<TRANSFORMS, simd_exec, simd_exec>
+    std::tuple<ExecInfo<TRANSFORMS, seq_exec, seq_exec>
+               //,ExecInfo<TRANSFORMS, seq_exec, simd_exec> //Need to fix... 
```

Comment:
@artv3 please add a JIRA issue for this so we don't forget it.

Reply:
@rhornung67, Do I need to be added on the JIRA page? I don't see RAJA under projects.